packages/web3-core/types/index.d.ts

```diff
@@ -200,9 +200,25 @@ export interface Log {
     blockNumber: number;
 }
 
-export interface Content {
-    pending: any;
-    queued: any;
+export interface TxPoolContent {
+    pending: TxPool;
+    queued: TxPool;
+}
+
+export interface TxPoolInspect {
+    pending: TxPool;
+    queued: TxPool;
+}
+
+export interface TxPool {
+    [address: string]: {
+        [nonce: number]: string[] | Transaction[];
```

Comment:
this should be only Transaction[ ].

Reply:
Not in all cases. I've combined two interfaces to one.